openmrs/omod/src/main/java/org/openmrs/projectbuendia/servlet/DataExportServlet.java

```diff
@@ -158,25 +165,15 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         List<Patient> patients = new ArrayList<>(patientService.getAllPatients());
         Collections.sort(patients, PATIENT_COMPARATOR);
 
-        // We may want to get the observations displayed in the chart/xform, in which case there
-        // are a few
-        // sensible orders:
-        // 1: UUID
-        // 2: Order in chart
-        // 3: Order in Xform
-
-        // Order in Xform/chart is not good as stuff changes every time we change xform
-        // So instead we will use UUID order, but use the Chart form to use the concepts to display.
```

Comment:
@dancunningham The above lines describe the previous column order but with this PR now the ordering is exact like the chart.